<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Mono</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ph_loginPage">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 12pt &quot;DejaVu Sans&quot;;color:rgb(255, 255, 255);
background-color:rgb(174,0,0);</string>
    </property>
    <property name="text">
     <string>Login In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>661</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;DejaVu Sans&quot;; color:rgb(174,0,0)</string>
    </property>
    <property name="text">
     <string>Bilgi İşlem Daire Başkanlığı</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ph_createAccountPage">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>600</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 12pt &quot;DejaVu Sans&quot;;color:rgb(255, 255, 255);
background-color:rgb(174,0,0);</string>
    </property>
    <property name="text">
     <string>Create A New Account</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_firat">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;DejaVu Sans&quot;;color:rgb(174,0,0)</string>
    </property>
    <property name="text">
     <string>Sing in or a new create account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ph_who">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>690</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 12pt &quot;DejaVu Sans&quot;;color:rgb(255, 255, 255);
background-color:rgb(174,0,0);</string>
    </property>
    <property name="text">
     <string>Who Person</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
